packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,121 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationToast
+
+<Status.Stable />
+
+<Story id="notification-notificationtoast--base" />
+<Props />
+
+The notification toast component non-disruptively provides quick and contextual feedback to the user. It can be based on the outcome of an action (i.e. saving a form) or it can pop up without having been triggered explicitly (i.e. a toast message "You made a new sale" showing real time sales in an app).
+
+## When to use it
+
+The notification toast should be used for a feedback on micro actions (i.e. successfully/unsuccessfully saving some information).
+
+## Usage guidelines
+
+- Be concise, specific and short in the message and always provide a body copy. If needed an optional headline can be included above the body copy.
+- The notification toast floats above the content, positioned at the bottom of the screen, centralized horizontally within the content area.
+- For multiple stacked toasts, the new toast slides on top of the previous one. The timeout of the stacked toasts follows chronological order.
+- The auto-dismiss duration is adjustable (default minimum duration is 3000 ms).
+- Toasts can also be dismissed by clicking on a close button in the top-right corner.
+
+## Variants
+
+<Story id="notification-notificationtoast--variants" />
+
+### Info
+
+Use the 'Info' variant for informational, neutral messages.
+
+### Confirm
+
+Use the 'Confirm' variant the successful messages.
+
+### Notify
+
+Use the 'Notify' variant for warning messages, and other information a user should be aware of.
+
+### Alert
+
+Use the 'Alert' variant for error messages.
+
+## Usage in code
+
+First, wrap your application in the `ToastProvider` which keeps track of the open toasts and ensures the accessibility of the toast.
+
+```tsx
+// _app.tsx for Next.js or App.js for CRA
+import { ToastProvider } from '@sumup/circuit-ui';
+
+export default function App() {
+  return <ToastProvider>{/* Your content goes here... */}</ToastProvider>;
+}
+```
+
+Then, use the `useNotificationToast` hook to open a toast from a component:
+
+```tsx
+import { useNotificationToast, Button } from '@sumup/circuit-ui';
+
+export function App({}) {
+  const { setToast } = useNotificationToast();
+
+  const handleClick = () => {
+    setToast({
+      body: 'This is a toast message',
+    });
+  };
+
+  return <Button onClick={handleClick}>Open toast</Button>;
+}
+```
+
+## Accessibility
+
+By their nature as small quick messages, or interruptions, toasts map to [live regions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions).
+Live region must always exist on the page at render, before the toast is generated.
```

Comment:
For next time: a line break in markdown doesn't do anything, a paragraphs should be on a single line without line breaks 🙂 You can use "toggle word wrap" in VSCode to make it easier to read them though